### PR TITLE
Kgpe d16 flashrom fix

### PR DESCRIPTION
**Pull request by Tonux599**
Allows KGPE-D16 users to flash the BMC internally.

@MrChromebox please review also as makes modifications to your work. As laid out in the commit messages:

> * Flashrom was being fetched with git and was always using `master`
> * No patches were being applied (i.e. `0100-enable-kgpe-d16.patch` was being ignored).

### CODE REVIEW

**Comment by MrChromebox** [comment]
would want to CP/test, but LGTM at a glance

**Comment by tlaurion** [comment]
@MrChromebox any regression on your side?

**Comment by MrChromebox** [comment]
@tlaurion will test tomorrow and approve once done

**Review by MrChromebox — APPROVED** [review]

**Pull request merged**
